### ISSUE TITLE
Updated Cyvcf2 Adds MNPs (Multiple Nucleotide Polymorphism) -Will Break Tests

### ISSUE THREAD

**Issue by mikaell** [Clinical-Genomics/scout#3092]
Somewhere between our locked version of Cyvcf2 (<0.10.0) and the latest version, support for MNPs was added. Displayed below is the result of basic content search.

### Newest
```
cyvcf2% grep -c mnp *
__init__.py:0
__main__.py:0
grep: __pycache__: Is a directory
cli.py:0
cyvcf2.c:36
cyvcf2.cpython-39-darwin.so:3
cyvcf2.pxd:0
cyvcf2.pyx:3
helpers.c:0
helpers.h:0
relatedness.h:0
grep: tests: Is a directory
```

### <0.10.0
```
cyvcf2% grep -c mnp *
__init__.py:0
__main__.py:0
grep: __pycache__: Is a directory
cli.py:0
cyvcf2.c:0
cyvcf2.cpython-38-darwin.so:0
cyvcf2.pxd:0
cyvcf2.pyx:0
helpers.c:0
helpers.h:0
relatedness.h:0
grep: tests: Is a directory
```

Running the test suite will eventually break on test `tests/adapter/mongo/test_query.py::test_query_snvs_by_coordinates` when calling the constant `CALLERS` with argument `mnp`. This because `variant.var_type` which previous was set to SNP or SNV will now may be set to MNP. I have come up with two simple solutions, but unsure which one is correct or best approach.

### A. Add Support for MNP
Add support for mnp:s in CALLERS constant. `/constants/__init__.py`. –I am unsure how this should look.

```
CALLERS = {
    "snv": [
        {"id": "gatk", "name": "GATK"},
        {"id": "freebayes", "name": "Freebayes"},
        {"id": "samtools", "name": "SAMtools"},
        {"id": "bcftools", "name": "Bcftools"},
        {"id": "deepvariant", "name": "DeepVariant"},
    ],
    
    "cancer": [
        {"id": "mutect", "name": "MuTect"},
        {"id": "pindel", "name": "Pindel"},
        {"id": "gatk", "name": "GATK"},
        {"id": "freebayes", "name": "Freebayes"},
    ],
    "cancer_sv": [
        {"id": "manta", "name": "Manta"},
        {"id": "gatk", "name": "GATK"},
    ],
    "sv": [
        {"id": "gatk", "name": "GATK"},
        {"id": "cnvnator", "name": "CNVnator"},
        {"id": "delly", "name": "Delly"},
        {"id": "tiddit", "name": "TIDDIT"},
        {"id": "manta", "name": "Manta"},
    ],
    "str": [{"id": "expansionhunter", "name": "ExpansionHunter"}],
}
```

### B. Cast MNP:s to SNPs
To my understanding a MNP is a set of SNVs. Can they be treated as equivalent? In `parser/variant/variant.py` add the following:

```
    if not category:
        category = variant.var_type
        if category == "indel":
            category = "snv"
        if category == "snp":
            category = "snv"
        if category == "mnp":
            category = "snv"
```

**Comment by northwestwitch** [comment]
It will be a bug in the moment we de-freeze the CyVCF2 lib, it's not a bug at the moment, right @mikaell ?

**Comment by dnil** [comment]
Thats a little odd; I would have expected all MNVs to be INDELs if not being merged SNVs. But we’ll spotted!

**Comment by mikaell** [comment]
I just talked to @jemten regarding this issue. MNVs should be split up to SNVs before the variant data reaches Scout. 

### C Cast Exception Upon MNVs + Remove MNVs From Repo
If Scout does not support MNVs an exception is thrown together with reference to documentation and how to split the SNV. We also must remove MNVs from our repo so that the test suite will pass.

**Comment by mikaell** [comment]
> It will be a bug in the moment we de-freeze the CyVCF2 lib, it's not a bug at the moment, right @mikaell ?

No, not at the moment.

**Comment by mikaell** [comment]
MNVs are converted upstreams to indels. Therefor MNVs should not reach Scout-land. The failed tests are from Scout's test suite. It might be that the test data reflects something that never can become reality.

Cycvf2 will cast this to a MNV (from scout/demo/643594.clinical.vcf.gz):

```
20	33012529	.	TAA	AAG	123.37	PASS	AB=0.2;ABP=6.91895;AC=3;AF=0.5;AN=6;ANN=AAG|intron_varia [...]
```

**Comment by dnil** [comment]
Thanks for the chat! So to summarise:
Alt A is blocked at the moment, until e.g. https://github.com/Clinical-Genomics/scout/issues/1435
Alt B above looks fine and should just be done! C is all good, but perhaps we leave it at normalising the demo file unless there are issues with B. Seems no harm in having MNVs in the SNV category when we already map the INDELs there.

**Comment by northwestwitch** [comment]
This one was fixed by #3114. Closing!
